internal/pkg/cli/pipeline_deploy.go

```diff
@@ -114,7 +119,7 @@ func (o *deployPipelineOpts) Execute() error {
 	}
 	o.prog.Stop(log.Ssuccessf(fmtPipelineDeployResourcesComplete, color.HighlightUserInput(o.appName)))
 
-	// read pipeline manifest
+	// Read pipeline manifest.
 	data, err := o.ws.ReadPipelineManifest()
```

Comment:
Quick question: If there are multiple pipeline in the workspace, which pipeline manifest should we be reading?

Reply:
That will be added/addressed in a future PR. 😸